x-pack/plugins/reporting/public/management/report_info_button.tsx

```diff
@@ -168,21 +291,44 @@ class ReportInfoButtonUi extends Component<Props, State> {
       );
     }
 
+    let message = this.props.intl.formatMessage({
+      id: 'xpack.reporting.listing.table.reportInfoButtonTooltip',
+      defaultMessage: 'See report info',
```

Comment:
Previously this text used the ending  period:

- See report info and error message.
- See report info and warnings.

"See report info" is consistently used without the ending period.